src/zjs_i2c_handler.c

```diff
@@ -0,0 +1,106 @@
+#include "zjs_i2c_handler.h"
```

Comment:
Add copyright header and include 2016-2017 since you're pulling in code from the other file.